docs/docusaurus/docs/contributing/application_integration_support.md

```diff
@@ -0,0 +1,137 @@
+
+| GX Cloud                    | GX OSS                          |
+|-----------------------------|---------------------------------|
+| - N/A                       | - 3.8 to 3.11                   | 
+
+### GX versions
+
+The following table defines the GX versions supported by GX Cloud and GX OSS.
+
+| GX Cloud                    | GX OSS                          |
+|-----------------------------|---------------------------------|
+| - N/A                       | - 0.17<br/> - 0.18              | 
+
+### Integrations
+
+The following table defines the supported GX Cloud and GX OSS integrations.
+
+
+| Application Type                         | GX Cloud             | GX OSS          |
+|------------------------------------------|----------------------|-----------------|
+| Data Source                              | - Snowflake          | - Snowflake<br/> - Generic SQL ¹               |
+| Configuration Stores                     | - N/A                | - Filesystem<br/> - AWS S3GCS Buckets<br/> - Azure Blob Store                |
+| Communication (notifications)            | - N/A ²               | - Slack (local) ³<br/> - email (local)<br/> - Microsoft Teams (local)<br/> - PagerDuty (local)                 |
+| Orchestrator                              | - Airflow ⁴           | - Airflow ⁴       |
+
+
+¹ Connection strings for various SQL Data Sources are supported, but dialect-specific SQL commands are not.<br/>
+² Support for Slack, email, and Zapier are planned for future releases.<br/>
+³ When used with the GX OSS library.<br/>
+⁴ Other Orchestrator applications that execute Python code are not supported.
+
+### GX components
+
+The following table defines the GX components supported by GX Cloud and GX OSS.
+
+| Component                                | GX Cloud             | GX OSS          |
+|------------------------------------------|----------------------|-----------------|
+| Expectations                             | - See [Available Expectations](/docs/cloud/expectations/manage_expectations#available-expectations) ¹. | - See [Create Expectations](/docs/guides/expectations/expectations_lp) ¹.                |
+| GX Agent                                 | - All versions               | - N/A        |
+
+¹ The creation and execution of custom Expectations are supported, but inner queries are not.
+
+## Community supported
+
+The following integrated applications, operating systems, and programming languages are supported by the community.
+
+### Operating systems
+
+The following table defines the operating systems supported by the community.
+
+| GX Cloud                       | GX OSS                        |
+|--------------------------------|-------------------------------|
+| - Mac/Linux ¹<br/>- Windows ²  | - Mac/Linux<br/>- Windows ²   | 
+
+¹ Required to run the GX Agent.<br/>
+² Untested and unsupported by GX
+
+### Python versions
```

Comment:
We're not support previous versions at all. We should remove both Python versions and GX versions section.